code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -495,6 +495,24 @@
 	toxpwr = 0.5
 	taste_description = "bad cooking"
 
+/datum/reagent/toxin/condensed_cooking_oil
+	name = "Condensed Cooking Oil"
+	description = "Taste the consequences of your mistakes."
+	reagent_state = LIQUID
+	color = "#d6d6d8"
+	metabolization_rate = 0.25 * REAGENTS_METABOLISM
+	toxpwr = 0
+	taste_mult = -2
+	taste_description = "awful cooking"
+
+/datum/reagent/toxin/condensed_cooking_oil/on_mob_life(mob/living/carbon/M)
+	if(prob(15))
+		M.vomit()
+	else
```

Comment:
this else makes it a 34% chance of heart damage instead of 40%

Reply:
I am generally okay with that, as the heart damage is supposed to be a minor thing, and I didn't want for them to both vomit and take heart damage at the same time, as I thought that would be too mean.